ui/shared/components/page/AcceptPolicies.jsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import Cookies from 'js-cookie'
+import { Modal } from 'semantic-ui-react'
+import { connect } from 'react-redux'
+import { Route, Switch, Link } from 'react-router-dom'
+
+import { updateUserPolicies } from 'redux/rootReducer'
+import { getUser } from 'redux/selectors'
+import { BooleanCheckbox } from '../form/Inputs'
+import ReduxFormWrapper, { validators } from '../form/ReduxFormWrapper'
+
+
+const COOKIE_ACTIONS = [{
+  primary: true,
+  content: 'Accept',
+  onClick: () => {
+    Cookies.set('accepted_cookies', true)
+    window.location.href = window.location.href
+  },
+}]
+
+const AcceptCookies = () =>
+  Cookies.get('accepted_cookies') ||
```

Comment:
It causes an un-wanted string `true` on the top left of the screen as shown in the below picture.
![image](https://user-images.githubusercontent.com/33550059/98172812-0288b580-1ec0-11eb-9b11-46868fabf8a2.png)
If change it to `!Cookies.get('accepted_cookies') &&` then the string `true` will disappear.